front/lib/api/assistant/conversation.ts

```diff
@@ -1643,9 +1643,9 @@ async function* streamRunAgentEvents(
       case "agent_action_success":
         // Store action in database.
         if (event.action.type === "retrieval_action") {
-          await agentMessageRow.update({
```

Comment:
Removing the field now. Just thought we would keep it in the database for a few days just in case we missed something. Will address the other comments accordingly.

Reply:
My point is either we keep updating the field and keep it around or  we remove it and stop updating.

Keeping the field but not updating is almost the worst of both worlds because it prevents rollbacking easily?

Reply:
I'm all for keeping everything around until everything works and we don't have reads to it anymore.

Reply:
True. Was not sure what we wanted to do with that, which I expressed by commenting this code to get your feedback.
The backfill script is checking that all lines are matching, and worst case scenario we can always do a point in time recovery and get the data back. So  I am removing the field.

Here is the Metabase query that shows: 
- Potential [mistakes](https://metabase.dust.tt/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiIgIFNFTEVDVCBhbS5pZCBmcm9tIGFnZW50X21lc3NhZ2VzIGFtXG4gICAgICBJTk5FUiBKT0lOIGFnZW50X3JldHJpZXZhbF9hY3Rpb25zIGFyYyBPTiAoYW0uXCJhZ2VudFJldHJpZXZhbEFjdGlvbklkXCIgPSBhcmMuaWQpXG4gICAgICBXSEVSRSBhcmMuXCJhZ2VudE1lc3NhZ2VJZFwiIDw-IGFtLmlkIiwidGVtcGxhdGUtdGFncyI6e319LCJkYXRhYmFzZSI6NH0sImRpc3BsYXkiOiJ0YWJsZSIsInBhcmFtZXRlcnMiOltdLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fX0=) in the data backfilling.
- [Count](https://metabase.dust.tt/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjo0LCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjcwLCJhZ2dyZWdhdGlvbiI6W1siY291bnQiXV0sImZpbHRlciI6WyJpcy1udWxsIixbImZpZWxkIiwxNjYyLHsiYmFzZS10eXBlIjoidHlwZS9JbnRlZ2VyIn1dXX19LCJkaXNwbGF5Ijoic2NhbGFyIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319) of `AgentRetrievalAction` with a null `agentMessageId` (migration in progress as I write)

Reply:
Also this PR can't be deployed before we are done with the backfilling migration.